app/script/util/confirm.js

```diff
@@ -50,7 +50,7 @@ $.fn.confirm = function(config) {
 
   const confirm = parent.find('.confirm');
   const group = parent.find('.participants-group');
-  let is_visible = true;
+  let isVisible = true;
```

Comment:
Yes, I changed all the `inVisible` :)

Reply:
File is already gone on edge branch anyway.

Reply:
hehe.. :)